SnapshotTesting.xcodeproj/project.pbxproj

```diff
@@ -850,6 +850,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```

Comment:
Is this project file change needed or did it sneak in?

We use [XcodeGen](https://www.github.com/yonaskolb/XcodeGen) to automatically generate our Xcode project file from a [project.yml](https://github.com/pointfreeco/swift-snapshot-testing/blob/master/project.yml) file. Can you make the changes there instead, and then run `swift run xcodegen` from the root of the repo to regenerate the project file?